pages/tokenDetails.tsx

```diff
@@ -1,15 +1,18 @@
+import { useState } from 'react'
 import PrimaryButton from '@/components/buttons/PrimaryButton'
 import SecondaryButton from '@/components/buttons/SecondaryButton'
 import Container from '@/components/Container'
 import Input from '@/components/Input'
-import Steps from '@/components/steps/Steps'
+import { Steps } from '@/components/steps'
 
 export default function tokenDetails() {
+  const [activeStep, setActiveStep] = useState(4)
```

Comment:
In this case, we leave it to the page function to take care of tracking what is the active step.

We do that, as it's the page function that uses form, and so it's able to track what happened in all the forms and then set the current step accordingly by calling `setActiveStep` function.